Code/GraphMol/DetermineBonds/catch_tests.cpp

```diff
@@ -0,0 +1,124 @@
+#include "RDGeneral/test.h"
+#include "catch.hpp"
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/DetermineBonds/DetermineBonds.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/AddHs.cpp>
+#include <iostream>
+#include <fstream>
+
+using namespace RDKit;
+
+TEST_CASE("Determine Connectivity") {
```

Comment:
If you have any example molecules where the Hueckel and van der Waals methods produce different results, it would be good to include a test which uses one or more of those just to make sure that the right code is actually being called.

@jhjensen2 may have some examples if you don't.